test/cppsuite/test_harness/core/configuration.h

```diff
@@ -173,6 +181,100 @@ class configuration {
         return func(value);
     }
 
+    /*
```

Comment:
Add "private:" here as the following functions are internal to file

Reply:
We're already in private here, FYI.